taskcluster/requirements.in

```diff
@@ -1 +1 @@
-taskcluster-taskgraph>=4.2.0
+git+https://github.com/bhearsum/taskgraph@fetch-multiple-artifacts
```

Comment:
Ah, I guess this is why you disabled hashes :p 

Don't forget to revert this before landing.

Reply:
Yeah, that's right :). Will do.